packages/expo-updates/ios/EXUpdates/Events/UpdatesEventManager.swift

```diff
@@ -0,0 +1,19 @@
+//  Copyright © 2019 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+public protocol UpdatesEventManager {
+  /**
+   The AppContext from expo-modules-core.
+   This is optional, but required for expo-updates module events to work.
+   */
+  var appContext: AppContext? { get }
```

Comment:
```suggestion
  weak var appContext: AppContext? { get }
```

maybe weak reference in the protocol?

Reply:
weak can't be specified in protocols it seems. that being said, I forgot to add it to the NoOp implementation.